cockatrice/src/tab_deck_editor.cpp

```diff
@@ -786,39 +785,11 @@ void TabDeckEditor::filterRemove(QAction *action) {
     filterModel->removeRow(idx.row(), idx.parent());
 }
 
-void TabDeckEditor::checkUnknownSets()
+void TabDeckEditor::checkFirstRunDetected()
 {
-    SetList sets = db->getSetList();
-
-    // no set is enabled. Probably this is the first time running trice
-    if(!sets.getEnabledSetsNum())
+    if(db->hasDetectedFirstRun())
     {
-        sets.guessSortKeys();
-        sets.sortByKey();
-        sets.enableAll();
-        db->emitCardListChanged();
-
+        QMessageBox::information(this, tr("Welcome"), tr("Hi! Its seems like it's the first time you run this version of Cockatrice.<br/>All the sets in the card database have been enabled; if you want to hide a set from the deck editor, disable it in the \"Edit Sets\" window."));
```

Comment:
\n over <br>?
At least that's what I've seen around in the client so far.
It gets recognized better by Transifex as well!

Will this message dialog shows up directly after the first app launch? Right before the `Edit sets...` pops up?

Not sure about the last part of the last sentence...
If you add detailed instructions/notes/information about how disabling+sorting sets effect the app in the `Edit sets...` dialog, that might be enough.
`"(...)\n All the sets in the card database have been enabled. Read more about changing set order or disabling specific sets in the the \"Edit Sets\" window."`